cumulus_library_umls/manifest.toml

```diff
@@ -3,7 +3,8 @@ study_prefix = "umls"
 [table_builder_config]
 file_names = [
     "umls_builder.py",
-    "static_builder.py"
+    "static_builder.py",
+    "ancilary_tables.sql"
```

Comment:
You can treat yourself to a trailing newline. JSON is the only weirdly cruel format.

Reply:
there is a linter for cargo.toml files, let's just switch to rust!